cypress/fixtures/add-delta-meter-read.js

```diff
@@ -0,0 +1,45 @@
+[
+    {
+      action: 'click',
+      selector: '[test-id="meter-read"]',
+    },
+  
+    {
+      action: 'click',
+      selector: '[test-id="add-meter-read"]',
+    },
+    {
+      action: 'selected',
+      selector: '[placeholder="Search External ID"]',
+      option: '[external-id-test-id]',
+    },
+    {
+      action: 'select-timezone',
+      selector: '[placeholder="Please Enter Timezone"]',
+      option: '[timezone-test-id]',
+    },
+    {
+      action: 'select',
+      selector: 'mat-select[formControlName="type"]',
+      option: '[read-type-test-id]',
+    },
+    {
+      action: 'select',
+      selector: 'mat-select[formControlName="unit"]',
+      option: '[unit-test-id]',
+    },
+    {
+      action: 'type',
+      selector: '[placeholder="Read value"]',
+      value: '100',
+    },
+    {
+      action: 'date-picker',
+      selector: '[test-id="choose-end-date"]'
```

Comment:
- Am wondering why did you remove the placeholder ?

Reply:
Replaced it with a test ID for improved future usability.